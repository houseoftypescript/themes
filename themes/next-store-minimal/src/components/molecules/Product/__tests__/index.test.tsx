import product01 from '@/assets/images/products/product01.jpeg';
import { render } from '@testing-library/react';
import { Product } from '../index';

describe('Product', () => {
  it('to match snapshot', () => {
    const { container } = render(
      <Product id="" title="" description="" image={product01} price={0} />
    );
    expect(container).toMatchSnapshot();
  });
});
