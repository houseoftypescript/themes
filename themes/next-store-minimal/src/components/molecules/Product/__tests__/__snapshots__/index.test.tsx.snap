// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product to match snapshot 1`] = `
<div>
  <div
    class="flex flex-col gap-4"
  >
    <div
      class="aspect-square bg-gray-100 border rounded overflow-hidden cursor-pointer hover:grow hover:shadow-lg transition-all duration-500"
    >
      <img
        alt=""
        class="w-full h-full"
        data-nimg="1"
        decoding="async"
        height="40"
        loading="lazy"
        src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
        srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
        style="color: transparent;"
        width="40"
      />
    </div>
    <div
      class="flex justify-between"
    >
      <h1
        class="text-gray-700"
      />
      <p
        class="font-medium"
      >
        $
        0
      </p>
    </div>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
      tabindex="0"
      type="button"
    >
      Buy Now
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
